androidsvg/src/main/java/com/caverock/androidsvg/SVGMapImageView.java

```diff
@@ -0,0 +1,49 @@
+package com.caverock.androidsvg;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.util.AttributeSet;
+import android.widget.ImageView;
+
+import org.jetbrains.annotations.Nullable;
+
+
+public class SVGMapImageView extends SVGImageView {
+
+    private SVG svg;
+
+    public SVGMapImageView(Context context) {
+        super(context);
+    }
+
+    public SVGMapImageView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    public SVGMapImageView(Context context, AttributeSet attrs, int defStyle) {
+        super(context, attrs, defStyle);
+    }
+
+    public SVG getSvg() {
+        return svg;
+    }
+
+    private void setSvg(SVG svg) {
+        this.svg = svg;
+    }
+
+    @Override
+    public void setSVG(SVG svg) {
+        if (svg == null) {
+            throw new IllegalArgumentException("SVG can not be null");
+        }
+
+        setSvg(svg);
```

Comment:
Is this function calling itself here? Not sure I follow this...

Reply:
Ohh, it's calling the super class' `setSvg()`.

Reply:
If it is called the superClass's `setSvg(SVG)` method, then it should say `super.setSvg(svg)`. This is calling itself. @dlupanciuc is that desired?

Reply:
The confusion is brought by naming(casing), it is initializing the SVG member variable. We might remove it for now since it is not being used.